praw/models/reddit/modmail.py

```diff
@@ -42,24 +42,29 @@ def _convert_user_summary(data, reddit):
                 key=lambda x: int(x.id, base=36), reverse=True)
 
     @classmethod
-    def parse(cls, data, reddit):
+    def parse(cls, data, reddit, convert_objects=True):
         """Return an instance of ModmailConversation from ``data``.
 
         :param data: The structured data.
         :param reddit: An instance of :class:`.Reddit`.
+        :param convert_objects: If True, convert message and mod action data
+            into objects (default: True).
 
         """
         conversation = data['conversation']
 
         conversation['authors'] = [reddit._objector.objectify(author)
                                    for author in conversation['authors']]
-        conversation['owner'] = reddit._objector.objectify(
-            conversation['owner'])
+        for entity in 'owner', 'participant':
+            conversation[entity] = reddit._objector.objectify(
+                conversation[entity])
 
-        if data['user']:
+        if data.get('user'):
```

Comment:
The reason it's done this way is that *internal* conversations, when fetched individually, have a `user` key that maps to an empty dictionary.